tests/integration/metadata/test_metadata.py

```diff
@@ -160,9 +161,17 @@ def test_detect_from_dataframes_multi_table_without_infer_sdtypes():
                     'billing_address': {'sdtype': 'unknown', 'pii': True},
                     'credit_card_number': {'sdtype': 'unknown', 'pii': True},
                 },
+                'primary_key': 'guest_email',
             },
         },
-        'relationships': [],
+        'relationships': [
+            {
+                'child_foreign_key': 'hotel_id',
+                'child_table_name': 'guests',
+                'parent_primary_key': 'hotel_id',
+                'parent_table_name': 'hotels',
+            }
+        ],
```

Comment:
Thanks to the changes, we can now detect the relationship in this case?

Reply:
Yeah because the hotel_id is now being detected as the primary key even if we don't detect the sdtypes, so it's able to find the relationship down the line